dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/validator/enrollment/SecurityOwnershipValidator.java

```diff
@@ -71,15 +69,15 @@ class SecurityOwnershipValidator implements Validator<Enrollment> {
   public void validate(Reporter reporter, TrackerBundle bundle, Enrollment enrollment) {
     TrackerImportStrategy strategy = bundle.getStrategy(enrollment);
     TrackerPreheat preheat = bundle.getPreheat();
-    User user = bundle.getUser();
+    UserDetails user = CurrentUserUtil.getCurrentUserDetails();
```

Comment:
Not sure if this call is expensive. Might also be a matter of style but now that we call this instead of passing it into methods we might as well call this in private methods 🤷🏻 If your not unit testing a function it saves a param of which we have many.

Reply:
Done.